### PR TITLE
feat(entity): add consumer for entity events in entity service #121

### PR DESCRIPTION
**Pull request by vraybaud**

### CODE REVIEW

File: entity-service/src/test/kotlin/com/egm/stellio/entity/service/EntitiesListenerTests.kt [hunk 1]
```diff
@@ -1,104 +1,93 @@
...
+                it.id == "urn:ngsi-ld:User:6ad19fe0-fc11-4024-85f2-931c6fa6f7e0"
             })
         }
-        confirmVerified(entityService)
+        confirmVerified()
+    }
+
+    @Test
+    fun `it should parse and transmit user deletion event`() {
+        val userCreateEvent = ClassPathResource("/ngsild/authorization/UserDeleteEvent.json")
```
Comment (HoucemKacem) [inline entity-service/src/test/kotlin/com/egm/stellio/entity/service/EntitiesListenerTests.kt:49]: you mean userDeleteEvent

File: entity-service/src/test/kotlin/com/egm/stellio/entity/service/EntitiesListenerTests.kt [hunk 2]
```diff
@@ -1,104 +1,93 @@
...
         }
-        confirmVerified(entityService)
+        confirmVerified()
     }
 
     @Test
-    fun `it should ignore badly formed observations`() {
-        val observation = ClassPathResource("/ngsild/aquac/Observation_missingAttributes.json")
+    fun `it should parse and transmit group membership deletion event`() {
+        val userCreateEvent = ClassPathResource("/ngsild/authorization/GroupMembershipDeleteEvent.json")
```
Comment (HoucemKacem) [inline entity-service/src/test/kotlin/com/egm/stellio/entity/service/EntitiesListenerTests.kt:80]: groupMembershipDeleteEvent

**Comment by HoucemKacem** [comment]
There are some indentation changes, you do it on purpose ?

**Comment by HoucemKacem** [comment]
There still some events such as grant/revoke access to entities for users, update users/groups roles.
Have we planned how to implement them ?

**Comment by vraybaud** [comment]
> There still some events such as grant/revoke access to entities for users
This is part of the Iteration 2
This will be done thanks to an API.

> update users/groups roles.
This is already handled thanks to append. (for further details, you can check : https://github.com/easy-global-market/keycloak-spi-kafka

**Pull request closed without merging**